src/components/Filters.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import '../App.css';
+import './style/Filters.css'
+import amsterdamRegular from '../img/filter-picker/amsterdam-regular.jpg';
+import amsterdamBlur from '../img/filter-picker/amsterdam-blur.png';
+import amsterdamBottomSobel from '../img/filter-picker/amsterdam-bottom-sobel.png';
+import amsterdamEmboss from '../img/filter-picker/amsterdam-emboss.png';
+import amsterdamLeftSobel from '../img/filter-picker/amsterdam-left-sobel.png';
+import amsterdamOutline from '../img/filter-picker/amsterdam-outline.png';
+import amsterdamRightSobel from '../img/filter-picker/amsterdam-right-sobel.png';
+import amsterdamSharpen from '../img/filter-picker/amsterdam-sharpen.png';
+import amsterdamTopSobel from '../img/filter-picker/amsterdam-top-sobel.png';
+
+
+
+class Filters extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: 'identity',
+        };
+    }
+
+    //val is string value
+    changeImage = val => {
+        const filterImage = document.getElementById("filter-applied");
+        const description = document.getElementById("img-description")
+        switch (val) {
+            case 'identity':
+                filterImage.src = amsterdamRegular;
+                description.innerHTML = "What do you think the identity filter does?";
```

Comment:
When you are writing a React webapp, you generally should never need to touch the DOM. Doing so defeats the purpose of React's [Virtual DOM (VDOM)](https://reactjs.org/docs/faq-internals.html). What's more, by reading `<input>` values directly, the valuable aspect of React's state being the single source of truth for a component is defeated. Consider, what would happen if one put multiple `<Filters />` on the page? Which element would `document.getElementById("img-description")` return?

Luckily, there's a quick way of fixing this by asking ourselves some questions about what React offers us. Instead of setting `innerHTML` based on an input, what if you instead set a state variable? Then, you could simply place your state variables inside of your `render()` function with a declaration like `<p>{this.state.currentDescription}</p>`! What's more, if we want to maintain the state as a single source of truth, we could bind a simple anonymous function to our input field that updates a separate state variable named something like `currentFilter`. This way, we avoid touching the DOM altogether, and leverage React to the best of our abilities.